### PR TITLE
Add VFS - Windows Storage Sense

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #124 ([Desktop] [VFS] Document native WinVFS for admins (Storage Sense))

Add Storage Sense to the VFS documentation.

Backport to 4.0 and 4.1

### CODE REVIEW

**Comment by michaelstingl** [comment]
@TheOneRing @erikjv @saw-jan Could we get your review please?

File: modules/ROOT/pages/vfs.adoc [hunk 1]
```diff
@@ -168,3 +168,15 @@ attrib -U +P .\* /D /S
 ----
 attrib +U -P .\* /D /S
 ----
+
+== Manage Drive Space With Storage Sense
+
+Microsoft provides a mechanism called `Storage Sense` which can automatically free up drive space for applications that use VFS. This is done by turing files that were set to be xref:vfs-states[locally available] back to cloud-based content, which are xref:vfs-states[placeholder files] on the local drive, requiring no space.
```
Comment (EParzefall) [inline modules/ROOT/pages/vfs.adoc:174]:
```suggestion
Microsoft provides a mechanism called `Storage Sense` which can automatically free up drive space for applications that use VFS. This is done by turning files that were set to be xref:vfs-states[locally available] back to cloud-based content, which are xref:vfs-states[placeholder files] on the local drive, requiring no space.
```

File: modules/ROOT/pages/vfs.adoc [hunk 2]
```diff
@@ -168,3 +168,15 @@ attrib -U +P .\* /D /S
...
+
+You can enable or disable Storage Sense via Windows menu:Settings[System > Storage > Storage]:
+
+image::vfs/vfs-settings-storage.png[Access storage configuration, width=400]
+
+and configure the general behaviour respectively each VFS relationship individually via the following section:
```
Comment (EParzefall) [inline modules/ROOT/pages/vfs.adoc:180]:
```suggestion
and configure the general behavior or each VFS relationship individually via the following section:
```

**Review by EParzefall — APPROVED** [review]

**Review by phil-davis — COMMENTED** [review]
English is OK

**Review by saw-jan — COMMENTED** [review]
LGTM

**Pull request merged**